projects/steering/test/test_steering.c

```diff
@@ -0,0 +1,109 @@
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include "adc.h"
+#include "adc_periodic_reader.h"
+#include "can.h"
+#include "can_msg_defs.h"
+#include "delay.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt_def.h"
+#include "misc.h"
+#include "ms_test_helpers.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
+#include "steering_control_stalk.h"
+#include "steering_digital_input.h"
+#include "steering_events.h"
+#include "test_helpers.h"
+#include "wait.h"
+

```

Comment:
are you testing all of your board here?

Reply:
or is it digital input?

Reply:
have one test file per module: test_steering_digital_input, and test_steering_stalk